accord-maelstrom/src/main/java/accord/maelstrom/Cluster.java

```diff
@@ -344,4 +349,28 @@ public static void run(Id[] nodes, QueueSupplier queueSupplier, Consumer<Packet>
             lookup.values().forEach(Node::shutdown);
         }
     }
+//
+//    public List<TxnId> waitingOn(String txnIdString)
```

Comment:
Remove these?

Reply:
These were incomplete attempts at implementing automation for much of my manual debugging process. Should have done it a while back and might have saved myself a day or two all told. But I'll do that at a later date.